examples/helloworld/dub.sdl

```diff
@@ -3,7 +3,5 @@ description "imgui hello world"
 authors "Luna"
 copyright "Copyright © 2020, Luna"
 license "BSL-1.0"
-dependency "bindbc-imgui" version="~>1.0.0"
-dependency "bindbc-sdl" version="~>0.19.1"
-dependency "bindbc-opengl" version="~>0.13.0"
```

Comment:
Huh, how does the sample work without sdl and opengl? Is it because they're transitive deps from imgui?

Reply:
I strictly believe so. At least, I never had to explicitly depend on `bindbc-sdl` or `bindbc-opengl` to make imgui work